deploy/cloudbuild.yaml

```diff
@@ -8,10 +8,16 @@ steps:
            "-t", "gcr.io/kritis-project/kritis-server:latest", "."]
   # Build preinstall container
   - name: "gcr.io/cloud-builders/docker"
-    args: ["build", "-f", "preinstall/Dockerfile",
+    args: ["build", "-f", "helm-hook/preinstall/Dockerfile",
            "-t", "gcr.io/kritis-project/preinstall:${COMMIT_SHA}", "."]
   - name: "gcr.io/cloud-builders/docker"
     args: ["tag", "gcr.io/kritis-project/preinstall:${COMMIT_SHA}", "gcr.io/kritis-project/preinstall:latest"]
+  # Build postinstall container
+  - name: "gcr.io/cloud-builders/docker"
+    args: ["build", "-f", "helm-hook/postinstall/Dockerfile",
+           "-t", "gcr.io/kritis-project/postinstall:${COMMIT_SHA}", "."]
+  - name: "gcr.io/cloud-builders/docker"
+    args: ["tag", "gcr.io/kritis-project/postinstall:${COMMIT_SHA}", "gcr.io/kritis-project/postinstall:latest"]
   # Build container to build resolve-tags
```

Comment:
Same as above, we should release the predelete image as well :)

Reply:
done